### ISSUE TITLE
Error while running anova_all: "ValueError: shapes (5,11) and (10,) not aligned: 11 (dim 1) != 10 (dim 0)"

### ISSUE THREAD

**Issue by elisabeth-chen** [CancerRxGene/gdsctools#155]
I receive the error pasted below when I try to use anova_all using the data frames attached. 

[test_gf_input.zip](https://github.com/CancerRxGene/gdsctools/files/541390/test_gf_input.zip)

```

In [2]: an_results = an.anova_all()
 [---------        24%                  ] 85 of 346 complete in 0.0 sec---------
------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-2-387a4692eada> in <module>()
----> 1 an_results = an.anova_all()

c:\users\ec12\appdata\local\continuum\anaconda3\lib\site-packages\gdsctools-0.14
.0-py3.5.egg\gdsctools\anova.py in anova_all(self, animate, drugs, multicore)
    839                 else:
    840                     res = self.anova_one_drug(drug_name, animate=False,
--> 841                                           output='dataframe')
    842                     self.individual_anova[drug_name] = res
    843                 if animate is True:

c:\users\ec12\appdata\local\continuum\anaconda3\lib\site-packages\gdsctools-0.14
.0-py3.5.egg\gdsctools\anova.py in anova_one_drug(self, drug_id, animate, output
)
    758             # Instead, we require dictionaries
    759             this = self.anova_one_drug_one_feature(drug_id, feature,
--> 760                     production=True)
    761             if this['ANOVA_FEATURE_pval'] is not None:
    762                 res[feature] = this

c:\users\ec12\appdata\local\continuum\anaconda3\lib\site-packages\gdsctools-0.14
.0-py3.5.egg\gdsctools\anova.py in anova_one_drug_one_feature(self, drug_id, fea
ture_name, show, production, directory)
    483
    484         self.anova_pvalues = self._get_anova_summary(self.data_lm,
--> 485                  output='dict')
    486
    487         # Store the pvalues. Note that some may be missing so we use try


c:\users\ec12\appdata\local\continuum\anaconda3\lib\site-packages\gdsctools-0.14
.0-py3.5.egg\gdsctools\anova.py in _get_anova_summary(self, data_lm, output)
    665         arr[0, 0] = 1                   # intercept
    666
--> 667         sum_sq = np.dot(arr, effects**2)[1:] # drop the intercep
    668         mean_sq = sum_sq / np.array(dof)
    669         Fvalues = mean_sq / (data_lm.ssr / data_lm.df_resid)

ValueError: shapes (5,11) and (10,) not aligned: 11 (dim 1) != 10 (dim 0)
```

**Comment by howard-lightfoot** [comment]
I had similar errors - "ValueError: shapes (5,11) and (10,) not aligned: 11 (dim 1) != 10 (dim 0)" - when running Pancan anova. In my case it could be fixed by removing the Media factor from the input genomic feature matrix (mobem). Setting include_media_factor = False didn't make a difference.

**Comment by cokelaer** [comment]
This seems to be related to a new version of Pandas (v0.19). I will update the code so that it can handle the older and newer version.

**Comment by cokelaer** [comment]
This should be fixed in v0.15.0
I have tried your data and it ran with success. Again, this was due to an updated version of pandas that and the code would work or not depending on the data rather than the trick of setting the media factor on/off.
